website/src/components/HomepageFeatures/index.tsx

```diff
@@ -46,10 +46,10 @@ const FeatureList: FeatureItem[] = [
   },
 ];
 
-const Feature = ({ title, Svg, description }: FeatureItem) => (
+const Feature = ({ title, img, description }: FeatureItem) => (
   <div className={clsx("col col--4")}>
     <div className="text--center">
-      <Svg className={styles.featureSvg} role="img" />
+      <img alt="none" className={styles.featureSvg} src={img} />
```

Comment:
It's better to have alt for images and not ignore them

```suggestion
      <img alt={alt} className={styles.featureSvg} src={img} />
```

Reply:
should be `title` right?

Reply:
If it describes the image, yes.